pages/home/EmailSignupForm.jsx

```diff
@@ -20,10 +20,10 @@ var PledgeSignupForm = React.createClass({
       return;
     }
 
-    if (process.env.NODE_ENV !== 'production' && !process.env.PLEDGE_MAILINGLIST_URL) {
+    if (process.env.NODE_ENV !== 'production' && !process.env.NEWSLETTER_MAILINGLIST_URL) {
       e.preventDefault();
-      alert("PLEDGE_MAILINGLIST_URL is not defined. Simulating a successful pledge signup now.");
-      window.location = "?pledge=thanks";
+      alert("NEWSLETTER_MAILINGLIST_URL is not defined. Simulating a successful newsletter signup now.");
+      window.location = "?signup=thanks";
```

Comment:
It's worth connecting with @alicoding on this.

This form will change to talk to ExactTarget rather than BSD now. Ali has been working on this for a number of other sites this week, so can help with the implementation.